[2014-05-06 02:05:09,216][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0xa58dad43, /10.0.1.4:55691 => /10.0.1.4:9301]], closing connection
java.io.IOException: Operation timed out
	at sun.nio.ch.FileDispatcher.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:21)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:198)
	at sun.nio.ch.IOUtil.read(IOUtil.java:166)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:245)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.read(NioWorker.java:64)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.process(AbstractNioWorker.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:318)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:89)
	at org.elasticsearch.common.netty.channel.socket.nio.NioWorker.run(NioWorker.java:178)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,009][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0xdc03cba9]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:699)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,010][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x19c998cc]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:699)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,011][INFO ][cluster.service          ] [Cannonball I] removed {[logstash-BelfordZ-Corvus.local-21777-2012][x9QVUji_QMS1zxTARNW6aA][BelfordZ-Corvus.local][inet[/10.0.1.4:9301]]{client=true, data=false},}, reason: zen-disco-node_failed([logstash-BelfordZ-Corvus.local-21777-2012][x9QVUji_QMS1zxTARNW6aA][BelfordZ-Corvus.local][inet[/10.0.1.4:9301]]{client=true, data=false}), reason transport disconnected (with verified connect)
[2014-05-06 03:04:26,012][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x8ba90600]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:702)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,013][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x223fd185]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:702)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,017][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x199cf5ff]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:702)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,021][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0xeaf515a5]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:705)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,024][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0xac7c52aa]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:705)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,026][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x7d07e0d6]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:705)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,028][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x78ef4bad]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:705)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,031][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x3bae00d8]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:705)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,037][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x77d0c14f]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:705)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,039][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0xdcec00ac]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:708)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 03:04:26,040][WARN ][transport.netty          ] [Cannonball I] exception caught on transport layer [[id: 0x66b34548]], closing connection
java.net.SocketException: Network is down
	at sun.nio.ch.Net.connect(Native Method)
	at sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:532)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.connect(NioClientSocketPipelineSink.java:108)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink.eventSunk(NioClientSocketPipelineSink.java:70)
	at org.elasticsearch.common.netty.channel.DefaultChannelPipeline.sendDownstream(DefaultChannelPipeline.java:574)
	at org.elasticsearch.common.netty.channel.Channels.connect(Channels.java:634)
	at org.elasticsearch.common.netty.channel.AbstractChannel.connect(AbstractChannel.java:207)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:229)
	at org.elasticsearch.common.netty.bootstrap.ClientBootstrap.connect(ClientBootstrap.java:182)
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:711)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:647)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:615)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:129)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:515)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[2014-05-06 12:43:51,257][INFO ][node                     ] [Cannonball I] stopping ...
[2014-05-06 12:43:51,300][INFO ][node                     ] [Cannonball I] stopped
[2014-05-06 12:43:51,300][INFO ][node                     ] [Cannonball I] closing ...
[2014-05-06 12:43:51,310][INFO ][node                     ] [Cannonball I] closed
[2014-05-06 12:44:10,542][INFO ][node                     ] [Thor] version[1.0.1], pid[22312], build[5c03844/2014-02-25T15:52:53Z]
[2014-05-06 12:44:10,542][INFO ][node                     ] [Thor] initializing ...
[2014-05-06 12:44:10,545][INFO ][plugins                  ] [Thor] loaded [], sites []
[2014-05-06 12:44:12,878][INFO ][node                     ] [Thor] initialized
[2014-05-06 12:44:12,878][INFO ][node                     ] [Thor] starting ...
[2014-05-06 12:44:12,972][INFO ][transport                ] [Thor] bound_address {inet[/0:0:0:0:0:0:0:0%0:9300]}, publish_address {inet[/192.168.0.101:9300]}
[2014-05-06 12:44:16,026][INFO ][cluster.service          ] [Thor] new_master [Thor][htiYY_OlQZ6q0A9mdTYUMg][BelfordZ-Corvus.local][inet[/192.168.0.101:9300]], reason: zen-disco-join (elected_as_master)
[2014-05-06 12:44:16,053][INFO ][discovery                ] [Thor] elasticsearch/htiYY_OlQZ6q0A9mdTYUMg
[2014-05-06 12:44:16,066][INFO ][http                     ] [Thor] bound_address {inet[/0:0:0:0:0:0:0:0%0:9200]}, publish_address {inet[/192.168.0.101:9200]}
[2014-05-06 12:44:16,080][INFO ][gateway                  ] [Thor] recovered [0] indices into cluster_state
[2014-05-06 12:44:16,081][INFO ][node                     ] [Thor] started
[2014-05-06 12:45:42,664][INFO ][cluster.service          ] [Thor] added {[logstash-BelfordZ-Corvus.local-22339-2012][imtW5SV0ST2BUEJ4isTA2Q][BelfordZ-Corvus.local][inet[/192.168.0.101:9301]]{client=true, data=false},}, reason: zen-disco-receive(join from node[[logstash-BelfordZ-Corvus.local-22339-2012][imtW5SV0ST2BUEJ4isTA2Q][BelfordZ-Corvus.local][inet[/192.168.0.101:9301]]{client=true, data=false}])
[2014-05-06 12:46:08,312][INFO ][cluster.metadata         ] [Thor] [logstash-2014.05.06] creating index, cause [auto(bulk api)], shards [5]/[1], mappings [_default_]
[2014-05-06 12:46:08,604][INFO ][cluster.metadata         ] [Thor] [logstash-2014.05.06] update_mapping [almanac] (dynamic)
[2014-05-06 12:46:08,852][INFO ][cluster.metadata         ] [Thor] [logstash-2014.05.06] update_mapping [almanac] (dynamic)
[2014-05-06 12:46:09,952][INFO ][cluster.metadata         ] [Thor] [logstash-2014.05.06] update_mapping [almanac] (dynamic)
[2014-05-06 12:46:14,914][INFO ][cluster.metadata         ] [Thor] [logstash-2014.05.06] update_mapping [almanac] (dynamic)
[2014-05-06 12:46:15,927][INFO ][cluster.metadata         ] [Thor] [logstash-2014.05.06] update_mapping [almanac] (dynamic)
[2014-05-06 15:21:26,429][INFO ][cluster.service          ] [Thor] removed {[logstash-BelfordZ-Corvus.local-22339-2012][imtW5SV0ST2BUEJ4isTA2Q][BelfordZ-Corvus.local][inet[/192.168.0.101:9301]]{client=true, data=false},}, reason: zen-disco-node_failed([logstash-BelfordZ-Corvus.local-22339-2012][imtW5SV0ST2BUEJ4isTA2Q][BelfordZ-Corvus.local][inet[/192.168.0.101:9301]]{client=true, data=false}), reason transport disconnected (with verified connect)
[2014-05-06 15:21:26,755][INFO ][node                     ] [Thor] stopping ...
[2014-05-06 15:21:26,778][INFO ][node                     ] [Thor] stopped
[2014-05-06 15:21:26,778][INFO ][node                     ] [Thor] closing ...
[2014-05-06 15:21:26,782][INFO ][node                     ] [Thor] closed
